encoding/csv/parse.ts

```diff
@@ -0,0 +1,116 @@
+// Copyright 2011 The Go Authors. All rights reserved. BSD license.
+// https://github.com/golang/go/blob/master/LICENSE
+// Copyright 2018-2022 the Deno authors. All rights reserved. MIT license.
+
+// This module is browser compatible.
+
+/** Port of the Go
+ * [encoding/csv](https://github.com/golang/go/blob/go1.12.5/src/encoding/csv/)
+ * library.
+ *
+ * @module
+ */
+
+import { assert } from "../../_util/asserts.ts";
+import type { ReadOptions } from "./_io.ts";
+import { Parser } from "./_parser.ts";
+
+export interface ParseOptions extends ReadOptions {
+  /**
+   * If you provide `skipFirstRow: true` and `columns`, the first line will be
+   * skipped.
+   * If you provide `skipFirstRow: true` but not `columns`, the first line will
+   * be skipped and used as header definitions.
+   */
+  skipFirstRow?: boolean;
+
+  /** List of names used for header definition. */
+  columns?: string[];
+}
+
+/**
+ * Csv parse helper to manipulate data.
+ * Provides an auto/custom mapper for columns.
+ *
+ * @example
+ * ```ts
+ * import { parse } from "https://deno.land/std@$STD_VERSION/encoding/csv/mod.ts";
```

Comment:
For imports within examples, it might be better to link to the most direct file. For example:
```ts
import { parse } from "https://deno.land/std@$STD_VERSION/encoding/csv/parse.ts";
```